src/components/Checkout/index.js

```diff
@@ -182,7 +186,10 @@ export const Checkout = (props) => {
 
     setPlacing(true)
     await onChangeSpot()
-    const result = await placeCart(cart.uuid, payload)
+    if (paymethodSelected?.paymethod?.gateway === 'credomatic') {
```

Comment:
Don't hardcode this gateway, create an array and add it, in order to optimize components